k8s/mongodb-deployment.yml

```
@@ -10,25 +10,22 @@ spec:
       containers:
-      - name: chatapp-mongodb
-        image: mongo:latest
-        ports:
-        - containerPort: 27017
-        env:
-        - name: MONGO_INITDB_ROOT_USERNAME
-          value: mongoadmin
-        - name: MONGO_INITDB_ROOT_PASSWORD
-          value: secret
+        - name: chatapp-mongodb
+          image: mongo:latest 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)